File: WinForms/DevExpress.WindowsMailClient.Win/VB/DevExpress.WindowsMailClient.Win/Forms/EmailSignatureEditForm.vb
Imports DevExpress.XtraEditors
Imports System

Namespace DevExpress.WindowsMailClient.Win.Forms

    Public Partial Class EmailSignatureEditForm
        Inherits XtraForm

        Public Sub New()
            InitializeComponent()
            toggleSwitch1.EditValue = CBool(Utils.UseAsEmailSignature)
            signatureEditFormControl1.LoadText(Utils.StartMhtText)
        End Sub

        Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs)
            Close()
        End Sub

        Private Sub btnSave_Click(ByVal sender As Object, ByVal e As EventArgs)
            Utils.StartMhtText = signatureEditFormControl1.GetText()
            Close()
        End Sub

        Private Sub toggleSwitch1_EditValueChanged(ByVal sender As Object, ByVal e As EventArgs)
            Utils.UseAsEmailSignature = CBool(CType(sender, ToggleSwitch).EditValue)
        End Sub
    End Class
End Namespace

File: WinForms/DevExpress.WindowsMailClient.Win/CS/DevExpress.WindowsMailClient.Win/Forms/EmailSignatureEditForm.cs
using System;
using DevExpress.XtraEditors;

namespace DevExpress.WindowsMailClient.Win.Forms {
    public partial class EmailSignatureEditForm : XtraForm {
        public EmailSignatureEditForm() {
            InitializeComponent();
            toggleSwitch1.EditValue = (bool)Utils.Utils.UseAsEmailSignature;
            signatureEditFormControl1.LoadText(Utils.Utils.StartMhtText);
        }

        void btnClose_Click(object sender, EventArgs e) {
            Close();
        }

        void btnSave_Click(object sender, EventArgs e) {
            Utils.Utils.StartMhtText = signatureEditFormControl1.GetText();
            Close();
        }

        void toggleSwitch1_EditValueChanged(object sender, EventArgs e) {
            Utils.Utils.UseAsEmailSignature = (bool)((ToggleSwitch)sender).EditValue;
        }
    }
}
